Generated code contains redeclared fields

Our schema.prisma file produces redeclared fields in the generated code. Here's a simple schema.prisma file that recreates the issue:

```
datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

generator client {
  provider = "go run github.com/prisma/prisma-client-go"
}

model AnnotationSet {
  id        Int      @id @default(autoincrement())
  createdAt DateTime @default(now())

  annotations Annotation[]
}

model Annotation {
  id        Int      @id @default(autoincrement())
  createdAt DateTime @default(now())

  annotationSet AnnotationSet @relation(fields: [setId], references: [id])
  setId         Int
}
```

Compiling with the generated code yields this error:

```go
db/db_gen.go:616:6: annotationSetParam redeclared in this block
	previous declaration at db/db_gen.go:67:6
db/db_gen.go:622:29: annotationSetParam.field redeclared in this block
	previous declaration at db/db_gen.go:72:
```

Go version `go1.16.2 darwin/amd64` and prisma-client-go version `v.0.8.0`. 

Update: the error is resolved when the `AnnotationSet` model is renamed to something else.

--- a/generator/templates/actions/actions.gotpl
+++ b/generator/templates/actions/actions.gotpl
@@ -33,20 +33,20 @@ var countOutput = []builder.Output{
 		{{ $model.Name.GoLowerCase }}Model()
 	}
 
-	type {{ $name }}Param struct {
+	type {{ $name }}DefaultParam struct {
 		data builder.Field
 		query builder.Query
 	}
 
-	func (p {{ $name }}Param) field() builder.Field {
+	func (p {{ $name }}DefaultParam) field() builder.Field {
 		return p.data
 	}
 
-	func (p {{ $name }}Param) getQuery() builder.Query {
+	func (p {{ $name }}DefaultParam) getQuery() builder.Query {
 		return p.query
 	}
 
-	func (p {{ $name }}Param) {{ $model.Name.GoLowerCase }}Model() {}
+	func (p {{ $name }}DefaultParam) {{ $model.Name.GoLowerCase }}Model() {}
 
 	type i{{ $model.Name.GoCase }}OrderByParam interface {
 		field() builder.Field

--- a/generator/templates/query.gotpl
+++ b/generator/templates/query.gotpl
@@ -30,14 +30,14 @@
 	}
 
 	{{ range $op := $.DMMF.Operators }}
-		func ({{ $nsQuery }}) {{ $op.Name }}(params ...{{ $nameUpper }}WhereParam) {{ $name }}Param {
+		func ({{ $nsQuery }}) {{ $op.Name }}(params ...{{ $nameUpper }}WhereParam) {{ $name }}DefaultParam {
 			var fields []builder.Field
 
 			for _, q := range params {
 				fields = append(fields, q.field())
 			}
 
-			return {{ $name }}Param{
+			return {{ $name }}DefaultParam{
 				data: builder.Field{
 					Name:     "{{ $op.Action }}",
 					List:     true,
@@ -93,14 +93,14 @@
 				// @{{ if $field.IsRequired }}required{{ else }}optional{{ end }}
 				func ({{ $nsQuery }}{{ $field.Name.GoCase }}Relations) {{ $method.Name }}(
 					params ...{{ $field.Type.GoCase }}WhereParam,
-				) {{ $name }}Param {
+				) {{ $name }}DefaultParam {
 					var fields []builder.Field
 
 					for _, q := range params {
 						fields = append(fields, q.field())
 					}
 
-					return {{ $name }}Param{
+					return {{ $name }}DefaultParam{
 						data: builder.Field{
 							Name:   "{{ $field.Name }}",
 							Fields: []builder.Field{
@@ -331,7 +331,7 @@
 		{{ if or ($field.IsID) ($field.IsUnique) }}
 			{{ $returnStruct = (print $name "ParamUnique") }}
 		{{ else }}
-			{{ $returnStruct = (print $name "Param") }}
+			{{ $returnStruct = (print $name "DefaultParam") }}
 		{{ end }}
 
 		{{ if $field.Kind.IncludeInStruct }}
@@ -424,8 +424,8 @@
 				}
 			{{ end }}
 
-			func (r {{ $struct }}) Order(direction types.Direction) {{ $name }}Param {
-				return {{ $name }}Param{
+			func (r {{ $struct }}) Order(direction types.Direction) {{ $name }}DefaultParam {
+				return {{ $name }}DefaultParam{
 					data: builder.Field{
 						Name:  "{{ $field.Name }}",
 						Value: direction,

